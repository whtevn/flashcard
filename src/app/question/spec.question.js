import renderer from 'react-test-renderer';
import React from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import FCD_Question from './component.question.js';

test('Flash card question renders correctly', () => {
  const tree = renderer.create(
      <MuiThemeProvider>
        <FCD_Question message="question" />
      </MuiThemeProvider>
      ).toJSON();
  expect(tree).toMatchSnapshot();
});
