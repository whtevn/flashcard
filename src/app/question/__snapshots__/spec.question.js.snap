// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flash card answer renders correctly 1`] = `
<div>
  question
</div>
`;
